hscontrol/types/config.go

```diff
@@ -935,6 +941,8 @@ func LoadServerConfig() (*Config, error) {
 			NodeMapSessionBufferedChanSize: viper.GetInt(
 				"tuning.node_mapsession_buffered_chan_size",
 			),
+			NodeRegistrationCacheExpiration: viper.GetDuration("tuning.node_registration_cache_expiration"),
+			NodeRegistrationCacheCleanup:    viper.GetDuration("tuning.node_registration_cache_cleanup"),
```

Comment:
I think these should be "nestable" in yaml, so 
```suggestion
			NodeRegistrationCacheExpiration: viper.GetDuration("tuning.node_registration_cache.expiration"),
			NodeRegistrationCacheCleanup:    viper.GetDuration("tuning.node_registration_cache.cleanup"),
```

but its fine to keep them as fields in go.